src/main/res/values/strings.xml

```diff
@@ -749,33 +749,33 @@
 
     <!-- 2016-04-03 -->
     <string name="error_tor_not_installed">Tor will not work unless Orbot is installed. Install now?</string>
-    <string name="error_tor_setting_failed">Failed to set proxy.</string>
+    <string name="error_tor_setting_failed">Could not set proxy.</string>
 
     <!-- 2016-04-10 -->
     <string name="error_parse_imgur_title">Parse Error</string>
-    <string name="error_parse_imgur_message">Imgur sent some data I couldn\'t understand.</string>
+    <string name="error_parse_imgur_message">Imgur sent incomprehensible data.</string>
 
     <string name="error_upload_fail_imgur_title">Upload failed</string>
-    <string name="error_upload_fail_imgur_message">The upload was rejected by Imgur for an unknown reason.</string>
+    <string name="error_upload_fail_imgur_message">Rejected by Imgur for unknown reason.</string>
```

Comment:
Unnecessarily terse, preferred the original

Reply:
The user knows it is trying to upload something. There is no need to suger-coat this non-descriptive error in non-descriptive language padding.

Reply:
I prefer padding out error messages if it makes them sound less cold/terse/ungrammatical